platform-hub-api/app/controllers/users_controller.rb

```diff
@@ -49,6 +58,42 @@ def revoke_admin
     head :no_content
   end
 
+  # POST /users/:id/activate
+  def activate
+    keycloak_agent_service.activate_user(@user)
+
+    @user.activate!
+
+    AuditService.log(
+      context: audit_context,
+      action: 'make_active',
+      auditable: @user
+    )
+
+    head :no_content
+  rescue => e
+    render_error "User activation failed - #{e.class}: #{e.message}",
+                :unprocessable_entity and return
+  end
+
+  # POST /users/:id/deactivate
+  def deactivate
+    keycloak_agent_service.deactivate_user(@user)
+
+    @user.deactivate!
```

Comment:
Also, when we deactivate a user, should we delete all their project memberships?